templates/ces_client/about.html

```diff
@@ -0,0 +1,74 @@
+                <h3 class='mt-4'>Connect to End Homelessness</h3>
+                <h4>A unified community response</h4>
+            </div>
+        </div>
+    </div>
+</div>
+
+<div class='row'>
+    <div class='col-10 offset-1 mb-3 pb-3'>
+        <h5><strong>About Coordinated Entry</strong></h5>
+        <p>Coordinated Entry is the front door to connect people facing homelessness to housing and supports in an accessible, equitable, transparent manner. This allows all partners to collaborate to improve access and outcomes for people without permanent housing while delivering trauma informed services.</p>
+        <p>People facing homelessness are asked to complete a Housing Assessment and linked to housing, if eligible, as opportunities become available.</p>
+
+        <h5 class='mt-5'><strong>Shelter Access in Cook County Suburbs</strong></h5>
+        <p>View shelter options and next steps <a href="http://www.suburbancook.org/emergency" target="_blank">by clicking here</a>.</p>
+
+        <h5 class='mt-5'><strong>Shelter Access In Chicago</strong></h5>
+        <p>Please visit any hospital emergency room or police station in Chicago, or call 311 and request transportation to shelter.</p>
+        <p>
+            <strong><i class="fas fa-user"></i> Single individuals can request shelter in person at the Chicago Garfield Service Center.</strong><br> 
+            10 S Kedzie Ave<br>
+            Chicago, IL 60612<br>
+            Monday through Friday, 9am – 5pm
+        </p>
+        <p>
+            <strong><i class="fas fa-users"></i> Families with children can request shelter at the Salvation Army Shield of Hope at any time.</strong><br> 
+            924 N Christiana Ave<br>
+            Chicago, IL 60651
+        </p>
+
+        <h5 class='mt-5'><strong>Useful Links</strong></h5>
+        <p><a href="https://www.csh.org/chicagoces" target="_blank">Chicago Coordinated Entry System</a></p>
+        <p><a href="https://allchicago.org" target="_blank">All Chicago: Making Homelessness History</a></p>
+        <p><a href="http://myentrypoint.org/" target="_blank">Suburban Coordinated Entry System</a></p>
+        <p><a href="http://www.suburbancook.org/" target="_blank">Alliance to End Homelessness in Suburban Cook County</a></p>
+
+        <h5 class="mt-5 mb-3"><strong>About the Bravely Engineered Navigator (B.E.N.)</strong></h5>
+        <div class='row'>
+            <div class='col'>
+                <p>B.E.N. represents a robot who knows a lot about the homeless experience. He helps personalize the survey-taking process. <a href="https://datamade.us/" target="_blank">DataMade (a civic technology company in Chicago)</a> designed B.E.N. and "Connect to End Homelessness" in partnership with the <a href="https://www.csh.org" target="_blank">Corporation for Supportive Housing (a non-profit that advances supportive housing solutions)</a>.</p>
+            </div>
+        </div>
+    </div>
+</div>
+
+<div class='row d-none d-sm-block'>
```

Comment:
The logos looked like they could be scaled appropriately in one row. If not, I'd keep them in stacked.